### PR TITLE
Total Game Score added

### PR DESCRIPTION
**Pull request by RobertsK284**
Still need to test some edge cases. Having trouble with try throw catch.

### CODE REVIEW

**Comment by RobertsK284** [comment]
Hi Alice. Would you please give feedback?

File: spec/gameSpec.js [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+    game = new Game();
+  });
+
+  it('can add a frame to a game', function() {
+    game.addFrame(3, 5);
+    expect(game.frames.length).toEqual(1);
+  });
+
+  it('sets the maximum number of normal frames to 9', function () {
+    var i = 0;
```
Comment (alicelieutier) [inline spec/gameSpec.js]: unused variable i

File: src/frame.js [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+Frame.prototype.getAllRolls = function () {
+  return this.rolls;
+};
+
+Frame.prototype._rollSum = function () {
+  if (this.rolls[2] == undefined) {
+    return this.rolls[0] + this.rolls[1];
+
+  } else {
+      return this.rolls.reduce(function (total, roll) {
```
Comment (alicelieutier) [inline src/frame.js]: Nice use of reduce.
You can also define a helper function called sum
```js
function sum(array){
    return array.reduce((a, b) => a + b)
}
```
and then call `sum(this.rolls)`
